bitnami/mean/requirements.yaml

```diff
@@ -1,4 +1,8 @@
 dependencies:
 - name: mongodb
   repository: https://kubernetes-charts.storage.googleapis.com/
-  version: 0.4.x
+  version: x.x.x
```

Comment:
I think it would be better to at least fix the major version of the dependency. Use 2.x.x